requirements.txt

```diff
@@ -9,3 +9,4 @@ numpy>=1.10.1
 Rtree>=0.8.3
 scipy>=0.14.1,!=0.19.1
 Shapely>=1.6.4
+importlib_metadata  # technically only required on python < 3.8; easier to install with conda across all versions
```

Comment:
Looks like this is not working in the `3.7` envs. But the failure is actually during environment creation, so maybe some other library is using this package and no longer getting the version it expected?

Reply:
Yeah, this turned out to be an apparent incompatibility between the `build` and `importlib_metadata` packages that is resolved by upgrading to the latest version of `build`, which is now distributed on `conda-forge` as `python-build`.